pkg/integrations/v4/logs/cfg.go

```diff
@@ -612,6 +613,10 @@ func newNROutput(cfg *config.LogForward) FBCfgOutput {
 		ValidateCerts:     cfg.ProxyCfg.ValidateCerts,
 	}
 
+	if cfg.IsFedramp {
+		ret.Endpoint = fedrampEndpoint
+	}
+
```

Comment:
IMO order of precedence would be: stg, fedramp and last euEndpoint.
So I would move 
```
  if license.IsRegionEU(cfg.License) {
    ret.Endpoint = euEndpoint			
  }
```
Should be first conditional.
(I don't know how to request a change 😿)